content/productivity/simultaneous-attack/plugin.js

```diff
@@ -2,7 +2,8 @@
 //
 // attack thing with many thing at the same time
 
-var pg = df.getProcgenUtils();
+import {getPlanetName} from "https://cdn.skypack.dev/@darkforest_eth/procedural";
+const pg = {getPlanetName: getPlanetName}
```

Comment:
I would rather suggest to use the `getPlanetName` function directly rather than wrapping it inside `pg`, which is an often used reference for the defunct `ProcgenUtils` this will just lead to confusion imo.